tests/Unit/Core/Util/ArrayFinderTest.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\Util;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Util\ArrayFinder;
+
+class ArrayFinderTest extends TestCase
```

Comment:
For this file, could you set the return value for methods ?

Reply:
I would like to know why this would be better.

What are the benefits of adding a void return type on a test function inside a PHPUnit TestCase?

As far as I know, return types
- are useful for auto-completion -> does not apply to test functions inside a PHPUnit TestCase as we don't call them
- are useful for type-checking -> does not apply to test functions inside a PHPUnit TestCase as we don't call them

Reply:
Done in https://github.com/matks/PrestaShop/commit/107e31a133288b35efeb0c6f3d72d5d2b73c8fa3